src/shim/WebAnimations.ts

```diff
@@ -116,5 +117,45 @@ export interface AnimationConstructor {
 	new (effect?: AnimationEffectReadOnly, timeline?: AnimationTimeline): Animation;
 }
 
-export const Animation = global.Animation as AnimationConstructor;
-export const KeyframeEffect = global.KeyframeEffect as KeyframeEffectConstructor;
+const _Animation = global.Animation as AnimationConstructor;
```

Comment:
Do you think it's possible to wrap all the mocking / replacement parts in a has check for `test`? At build time this should be then converted to `false` and then removed from the prod build?

Reply:
👍 I'll just need to make a change in cli-build-app then to set this flag for the test build.